sdk/python/kfp/dsl/for_loop.py

```diff
@@ -64,33 +77,46 @@ def _get_subvar_type(type_name: str) -> Optional[str]:
     return match['value_type'].lstrip().rstrip() if match else None
 
 
-class LoopArgument(pipeline_channel.PipelineParameterChannel):
-    """Represents the argument that are looped over in a ParallelFor loop.
+def _make_name(code: str):
+    """Makes a name for a loop argument from a unique code."""
+    return f'{LOOP_ITEM_PARAM_NAME_BASE}-{code}'
+
+
+def name_is_loop_argument(name: str) -> bool:
```

Comment:
Is this used anywhere?

Reply:
It's used in https://github.com/kubeflow/pipelines/blob/7bd31d104bd403a830bf2a455c9c2c0dbf493c4d/sdk/python/kfp/dsl/for_loop_test.py#L152

Reply:
Can we move this to be a function in the test module?